packages/block-editor/src/components/inspector-controls-tabs/list-view-tab.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { __experimentalUseSlotFills as useSlotFills } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import InspectorControls from '../inspector-controls';
+import InspectorControlsGroups from '../inspector-controls/groups';
+
+// This tab restricts the blocks that may render to it via the whitelist below.
+const whitelist = [ 'core/navigation' ];
```

Comment:
Shouldn't the block determine this tab to appear or not to appear? I think hard coding like this is not future proof.

Reply:
Just to restrict this a bit more while it's an experiment, otherwise anyone can use the slot (even though it's not a stable API).

Reply:
the term whitelist is best avoided... i'll update

Reply:
Good point, thanks for doing that 👍

Reply:
Thanks for cleaning that up 👍